pkg/kexec/kexec_linux_other.go

```diff
@@ -1,8 +1,8 @@
-// Copyright 2015-2017 the u-root Authors. All rights reserved
+// Copyright 2015-2019 the u-root Authors. All rights reserved
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build linux,!amd64
+// +build linux,!amd64,!arm
```

Comment:
but if it's named linux_other, and kexec_linux_amd64.go and kexec_linux_arm exist, why do you need this build tag? Or don't you just need the !amd64 and !arm?

Reply:
The `other` build tag doesn't have any special meaning to the compiler as far as I'm aware. This line prevents the compiler complaining about multiple definitions of `FileLoad`. It's a catch-all for architectures which aren't amd64 and aren't arm.

Alternatively, we can just delete this file and make it a compile-time error to use this package from an unsupported platform.